### DISCUSSION TITLE
本地收藏

### DISCUSSION THREAD

**Discussion by twtan** [xiaoyaocz/flutter_dmzj#133] [General]
现在打开任意一部漫画都会提示已加入本地收藏夹，而且没有重复检测打开几次提示几次
感觉本地收藏夹现在就是个本地浏览记录的功能

**Comment by xiaoyaocz** [comment]
设置中关闭"自动收藏神隐漫画"。
现在接口有点问题，日本都会被判定神隐漫画

**Comment by xiaoyaocz** [comment]
#138

**Comment by twtan** [comment]
我想说的是另一个问题，收藏没有重复检测，每次打开都会提示一次加入收藏，虽然并不影响使用

**Reply by xiaoyaocz** [reply]
#138 已解决，下版本更新
